build.gradle

```diff
@@ -19,7 +19,7 @@ allprojects {
     repositories {
         google()
         jcenter()
-        
+        maven { url 'https://jitpack.io' }
```

Comment:
a TODO to remove would be good so that we don't forget as it's only temporarily change for testing before 1.9 release

Reply:
done :)